data/json/items/gun/223.json

```diff
@@ -325,6 +350,53 @@
     "modes": [ [ "DEFAULT", "semi-auto", 1 ], [ "BURST", "3rd burst", 3 ] ],
     "magazines": [ [ "223", [ "stanag30", "stanag50", "survivor223mag" ] ] ]
   },
+  {
+    "id": "oa93",
+    "copy-from": "rifle_semi",
+    "looks_like": "skorpion_61",
+    "type": "GUN",
+    "name": { "str": "OA-93" },
+    "description": "An AR-15 derivative pistol manufactured by Olympic Arms in the nineties. The main difference compared to the AR-15 is that the recoil spring has been moved to the top of the gun, circumventing the necessity for a solid buttstock.",
+    "weight": "2023 g",
+    "volume": "2135 ml",
+    "price": 125000,
+    "to_hit": -2,
+    "bashing": 9,
+    "material": [ "steel", "aluminum" ],
+    "symbol": "(",
+    "color": "dark_gray",
+    "ammo": "223",
+    "range": -6,
+    "ranged_damage": -16,
```

Comment:
-11 is appropriate.
<https://ndiastorage.blob.core.usgovcloudapi.net/ndia/2010/armament/WednesdayCumberlandPhilipDater.pdf>
![image](https://user-images.githubusercontent.com/7764202/74091892-9db04400-4a82-11ea-8bba-3a71c044932c.png)